src/components/CompareResults/TableHeader.tsx

```diff
@@ -72,19 +75,21 @@ function FilterableColumn({
             theme.palette.mode == 'light'
               ? Colors.Background200
               : Colors.Background200Dark,
-          borderRadius: '4px',
-          fontSize: 'inherit',
+          borderRadius: 0.5,
+          padding: '6px 12px',
         })}
       >
         {name}
-        <FilterListIcon
-          fontSize='small'
-          color={hasFilteredValues ? 'primary' : 'inherit'}
-          sx={{ marginInlineStart: 1 }}
-          titleAccess={
-            hasFilteredValues ? 'Some filters are active' : 'No active filters'
-          }
-        />
+        <Box
+          sx={{
+            paddingInlineStart: 0.5,
+            color: hasFilteredValues ? 'primary' : 'inherit',
```

Comment:
Turns out that just `primary` doesn't work, we need `primary.main`. In the `FilterListIcon` (that is `SvgIcon`) like before, there's some logic to do that, but with  `Box` we need to be explicit.

```suggestion
            color: hasFilteredValues ? 'primary.main' : 'inherit',
```

Reply:
Also I wonder if we should just remove the condition to have it always blue:
```suggestion
            color: 'primary.main',
```

what do you think?

Reply:
Yes, that works. The condition isn't needed. Thanks!